onboarding.md

```diff
@@ -23,55 +23,39 @@ To facilitate your success, Casey will help you to plan your training, to devise
 
 **Deadlines:** Our lab has worked hard to develop a reputation for high-quality science that is well presented.
 We all benefit from this reputation, but we must also work to maintain it.
-In order to maintain the quality of our lab's output, we've established deadlines for various outputs.
-Each of these applies to sharing a complete version that the author deems ready for submission in the Greene Lab slack's `#general` channel.
+Abstracts for meetings must be shared with all co-authors, including Casey, at least one week prior to the deadline for submission.
+Failure to abide by this guideline will result in missing whatever the opportunity in question is.
 
-The specific deadlines for various types of outputs are:
-
-- Manuscripts must be shared two weeks before any deadlines.
-- Posters must be shared one week before the deadline for printing.
-- Scientific talks based on a submitted abstract must be shared one week before the presentation.
-- Meeting abstracts must be shared one week before the deadline for submission.
-
-Lab members are given a two-day period to provide feedback on the document.
-We expect that authors will then revise the document to incorporate feedback provided within the initial two-day period.
-Authors are encouraged but not required to address feedback received after the initial two days, as it may not always be practicable.
-
-This does not eliminate the need for all coauthors to approve a document.
-Coauthors are not required to provide their feedback within the two-day window.
-Coauthors can hold submission of a document until they approve; however, according to ICMJE guidelines extreme holds may result in a change from authorship to acknowledgement.
-
-In the case that all feedback received within the two-day period has been addressed and all coauthors approve, the submission can proceed.
-
-Failure to abide by these guidelines will result in missing whatever the opportunity in question is.
+Trainees in the lab will often receive opportunities to present their work at scientific conferences.
+These presentations reflect on the entire lab and must be presented to the research lab during a `braintrust` meeting prior to the conference, and lab members are expected to address feedback that is provided.
```

Comment:
There are no longer any references to non-conference manuscripts in this section. Does that mean that manuscript deadlines are on a case-by-case basis?

Reply:
The manuscripts section is gone because it was hard for me to come up with an appropriate rule for all manuscripts. It seemed easier for presentations / abstracts.